### ISSUE TITLE
Misdetect on iPhone11

### ISSUE THREAD

**Issue by KAZUMA87** [sindresorhus/devtools-detect#42]
Devtools-detect detects DevTools use in Mobile Safari or Chrome on iPhone 11.
But this is misdetect.Is there any solution?

**Comment by Directory** [comment]
no. and not just on iph11 but really any ios device. i guess safari does something strange with viewport sizes when the keyboard opens and you scroll around.

**Comment by christiaanwesterbeek** [comment]
Yep. Devtools will be detected as open if simply the URL bar becomes visible on iOS 16.1.1 on an iPhone 12 mini

![image](https://user-images.githubusercontent.com/465989/206438096-efbf2b9d-c8ab-43da-a758-441db3513f41.png)
